packages/asc-ui/src/components/Publication/Publication.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react/'
+import { action } from '@storybook/addon-actions'
+import { Column, CustomHTMLBlock, Row, BlogHeader } from '../../index'
+import Summary from '../Typography/Summary'
+import Downloader from '../Downloader'
+import Publication from './Publication'
+import publicationJSON from './publication_feed.json'
+import { BlogMetaList, BlogContent } from '../Blog'
+
+storiesOf('Composed/Blog/Publication', module).add('dataportal version', () => {
+  const {
+    title,
+    body,
+    field_publication_intro: intro,
+    field_file_size: fileSize,
+    field_file_type: fileType,
+  } = publicationJSON.data[0].attributes
+  const cmsRoot = 'https://acc.cms.data.amsterdam.nl'
```

Comment:
This will only work with govroam/vpn... Maybe use a JSON object?

Reply:
Agreed, also make it easier for yourself to just use a normalized json? We don't need to use a real-life example json :)

Reply:
done